public/iframe-selector/injected-script.js

```diff
@@ -91,7 +91,12 @@ function onMouseLeave(e) {
 }
 
 function onClick(e) {
-  var cssPath = finder(e.target);
+  let cssPath;
+  try {
+    cssPath = finder(e.target);
+  } catch (e) {
+    console.error(e);
```

Comment:
Is this an `error` or a `warn` if we don't consider a reason to fail? 🙂

Reply:
This is for the log line to stand out of the logs if such a case happen.
I want to be able to see if there are other case when it does not work.

I agree that this should not be considered an error anyway as it's handled so will change to `console.log`